src/components/SearchResultsBox.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import SearchResultsBox from './SearchResultsBox';
+
+describe('SearchResultsBox', () => {
+  it('renders loading screen when fetching', () => {
+    const wrapper = shallow(<SearchResultsBox fetchingInProgress />);
+    
+    expect(wrapper.find('center').exists()).toEqual(true);
+  });
+
+  it('renders message if no results', () => {
+    const wrapper = shallow(<SearchResultsBox results={{}} />);
+    expect(wrapper.contains([
+      <h4>No results were found, make sure the repo you're looking for exists and is spelled correctly.</h4>,
```

Comment:
Yo look at this line, this rendering via Github is weird. My guess is that the `'` is throwing things off. Try making the text inside a react text javascript string so we can avoid the single quote problem. I tested this out and it works. Change the line to:

```javascript
<h4>{ "No results were found, make sure the repo you're looking for exists and is spelled correctly." }</h4>,
```